altair/vegalite/v5/api.py

```diff
@@ -1104,24 +1104,31 @@ def to_html(
             **kwargs,
         )
 
-    def to_url(self, *, fullscreen: bool = False) -> str:
+    def to_url(self, *, open_browser: bool = True, fullscreen: bool = False) -> str:
```

Comment:
This technically breaks if someone used to rely on fullscreen as a positional argument via `to_url(True)`. I don't think it's a huge deal, but also have nothing against reversing the order.

Reply:
Good thing is that thanks to the `*`, `fullscreen` has always been a [keyword-only argument](https://peps.python.org/pep-3102/) so this should be fine.

Reply:
This is why we should lean towards kwarg-only APIs when we have the choice :) yay thanks past us

Reply:
This is cool, I haven't come across this use of `*` to block additional positional args. Thanks for linking the pep!